app/serializers/loan_question_serializer.rb

```diff
@@ -29,4 +29,8 @@ def optional
   def required_loan_types
     object.loan_question_requirements.map { |i| i.loan_type.label.to_s }
   end
+
+  def status
+    object.status ? object.status : "active"
```

Comment:
Equivalent but shorter and more conventional: `object.status || "active"`

BUT -- you need to make sure status can't be empty string ("") instead of nil. Because empty string evaluates to `true` when used in a condition. If that is a possibility, you can use `object.status.presence || "active"`. `.presence` returns `nil` if the value is `.empty?`, or just the value itself otherwise.